app/src/main/java/network/SingleApi.kt

```diff
@@ -11,9 +11,6 @@ interface SingleApi {
     @GET("log_in")
     fun logInWithContext(@Tag context: Context, @Header("x-bypass") flag: Int): Single<UserToken>
```

Comment:
The first parameter is `Context` sent as `retrofit2.http.Tag` supported in https://github.com/square/retrofit/pull/3028